Update ongoing maintenance list

ccostino opened the pull request:
This update adds a step to review all expiring certificates to make sure we do not miss any coming up in the near future.

Note that this is a temporary step until alerts are back in place for certificate expirations.

## Changes proposed in this pull request:
- Cleans up formatting at the top of the guide a bit
- Adds a small section for checking expiring certificates

:sunglasses: [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/ongoing-platform-maintenance-updates/docs/ops/maintenance-list/)

## Security Considerations
- Low/no risk as it is just reformatting information already on the page and noting another shell command to run.

ccostino committed:
Update ongoing maintenance list

This update adds a step to review all expiring certificates to make sure we do not miss any coming up in the near future.

Note that this is a temporary step until alerts are back in place for certificate expirations.

ccostino committed:
Removed extraneous note.

ChrisMcGowan commented on _docs/ops/maintenance-list.md:
```diff
@@ -66,6 +66,10 @@ Review the [detailed guide on customer support]({{site.baseurl}}/docs/ops/custom
+## Review expiring certificates
+
+From the jumpbox in each of our four environments, `tooling`, `development`, `staging`, and `production`, run `doomsday dashboard` to review all expiring certificates. Make note of any that are expiring within two weeks and make sure the team is aware.
+
```
You can also look at the GUI while on VPN: https://doomsday.fr.cloud.gov/

ccostino replied:
Ah, that's much easier, thank you!  I also see what you mean about Comic Sans... 😆

ccostino committed:
Merge branch 'master' into ongoing-platform-maintenance-updates

ccostino committed:
Added a link to the GUI dashboard

h/t to @ChrisMcGowan for the link and suggestion!  I also expanded on the steps to take a bit more.

ccostino committed:
Added note about the VPN for Prometheus

ccostino committed:
Merge branch 'master' into ongoing-platform-maintenance-updates
